content/en/shader-editor/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Shader Editor
+template: shader-editor-page.tmpl.html
+position: 0.5
+---
+
+Welcome to the __Shader Editor__ User Manual.
+
+Here you will find:
+- An [introduction][1] to the tool
+- A general [overview][3]
+- Information on the [window layout][2]
+- [Tutorials][4] explaining how to accomplish common tasks and
+- A [reference][5] which documents the various types of supported nodes.
+
```

Comment:
If this page will be linked to by tweets, HN, having an image/video/example project would be great!

Reply:
yup, can we do this in followup pr?

Reply:
It depends if we link to a blog post which will have some gifs/pics. If we do, then this can be in a future PR